cantor-archive/src/main/java/com/salesforce/cantor/archive/AbstractBaseArchiver.java

```diff
@@ -42,10 +48,10 @@ static ArchiveInputStream getArchiveInputStream(final Path archiveFile) throws I
         return new TarArchiveInputStream(new GzipCompressorInputStream(new BufferedInputStream(Files.newInputStream(archiveFile))));
     }
 
-    static void checkArchiveArguments(final Object instance, final String namespace, final Path destination) {
+    static void checkArchiveArguments(final Object instance, final String namespace, final Path destination) throws IOException {
         checkArgument(instance != null, "null cantor instance, can't archive");
         checkString(namespace, "null/empty namespace, can't archive");
-        checkArgument(Files.notExists(destination), "destination already exists, can't archive");
+        checkArgument(Files.notExists(destination) || Files.size(destination) == 0, "destination already exists and is not empty, can't archive");
```

Comment:
Did you mean `Files.exists(destination) && Files.size(destination) == 0`

Reply:
Actually no this is correct. It's checking to make sure the file isn't there or the file is empty. If you give it an empty file it'll just overwrite it, and if the file doesn't exist at all I create it.